nx.json

```diff
@@ -38,6 +38,11 @@
       "options": {
         "packageRoot": "dist/packages/{projectName}"
       }
+    },
```

Comment:
I'm not sure if this is expected or not. This entire PR is auto-generated with the library generator.

Reply:
yeah its not please revert https://github.com/microsoft/fluentui-contrib/issues/350